### ISSUE TITLE
In Holdings, visually distinguish displayed table in header

### ISSUE THREAD

**Issue by jenicek** [beancount/fava#1383]
On Holdings page, all headers in `.headerline` have the same text color, which harms navigation within that page. I would propose to visually distinguish the current table by a different color of header in `.headerline`, such as in the following patch.

```
diff --git a/frontend/css/base.css b/frontend/css/base.css
index 17466ad8..4829f835 100644
--- a/frontend/css/base.css
+++ b/frontend/css/base.css
@@ -311,7 +311,7 @@ button.round,
 }
 
 .headerline a {
-  color: var(--text-color-lighter);
+  color: var(--link-color);
 }
 
 .headerline a:hover {
```

Alternatively, I would propose to make the `--text-color-lighter` much lighter, so that it can be seen on a computer screen well.

**Comment by yagebu** [comment]
Yep, the contrast there is unsufficient. They are not quite the same color - takes a good and well-calibrated display to see the difference I guess :D

**Comment by yagebu** [comment]
Fixed in 9956c3b8e3b2d5a8734141fc44c1a243a37c7264
